optima/plotting.py

```diff
@@ -25,6 +25,40 @@
 
 
 
+def humanise_y_ticks(ax):
```

Comment:
good point. will fix. 

i have to pay extra attention as alas, there are different coding conventions within the javascript modules, in the server code, the optima code, and mine. :(